src/classes/VOL_SharedCode.cls

```diff
@@ -310,13 +310,8 @@ global with sharing class VOL_SharedCode {
                 if (contact.Department != null) con.Volunteer_Organization__c = contact.Department;
             }
 
-            if (UTIL_Describe.isGuestAccessElevated) {
-                UTIL_Describe.checkCreateAccess('Contact', setFlds);
-                DatabaseDml.getInstance().updateRecords(listCon, dmlDuplicateOptions);
-            } else {
-                UTIL_Describe.checkUpdateAccess('Contact', setFlds);
-                Database.update(listCon, dmlDuplicateOptions);
-            }
+            VOL_Access.getInstance().checkUpdateAccess('Contact', setFlds);
```

Comment:
Should we turn this into a variable with a getter rather than repeatedly requesting the instance?